Tests/ConnectionFactoryTest.php

```diff
@@ -71,7 +71,7 @@ public function testDefaultCharset(): void
         $connection    = $factory->createConnection($params);
 
         $this->assertInstanceof(FakeConnection::class, $connection);
-        $this->assertSame('utf8', $connection->getParams()['charset']);
+        $this->assertSame('utf8mb4', $connection->getParams()['charset']);
```

Comment:
Are you sure this is OK? It does not look OK to change that default value like that (I know it's a bad default, but it might result in a lot of migrations for a lot of people).

Reply:
The fake driver ist actually a mysqlplatform that is why it is changing: https://github.com/doctrine/DoctrineBundle/blob/6e578fd23ddb656af5c47c1014576490cebd4b0a/Tests/ConnectionFactoryTest.php#L156

Reply:
Before the check was on the driver not on the platform.

Reply:
:thinking: in fact, I think it might be fine, since people using MySQL ended up with `utf8mb4` before https://github.com/doctrine/DoctrineBundle/pull/1456